app/screens/thread/thread_post_list/thread_post_list.tsx

```diff
@@ -52,18 +59,20 @@ const ThreadPostList = ({
         }
     }, [isCRTEnabled, posts, rootPost, serverUrl, teamId, thread]);
 
+    const lastViewedAt = isCRTEnabled ? (thread?.viewedAt ?? 0) : channelLastViewedAt;
```

Comment:
For my own understanding:  `thread?.viewedAt ?? 0` => the 0 means that the **thread** was never viewed before

Reply:
Yes it was never viewed before.